<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="left_container">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="toolTipDuration">
      <number>4</number>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>BackDB</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>160</width>
              <height>440</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Page 1</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>User</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Game</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>UserGame</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>105</width>
              <height>423</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Page 2</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Projeto Pav 2023&lt;br/&gt;BackDB&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="main_container">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="top_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbpush">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Dados de BackDB</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="main_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pg_uhg">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QFrame" name="frame_13">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="5" column="0">
                 <widget class="QLineEdit" name="uhgreview">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Review</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="uhgid">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>ID</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="uhguserid">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>UserID</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLineEdit" name="uhgrate">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Nota</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <spacer name="verticalSpacer_12">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="2" column="0">
                 <widget class="QLineEdit" name="uhggameid">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>GameID</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_14">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <item>
                 <widget class="QPushButton" name="pbcuhg">
                  <property name="text">
                   <string>Cadastrar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbsuhg">
                  <property name="text">
                   <string>Selecionar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbauhg">
                  <property name="text">
                   <string>Atualizar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbeuhg">
                  <property name="text">
                   <string>Excluir</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_13">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pg_game">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QFrame" name="frame_11">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="5" column="0">
                 <widget class="QLineEdit" name="gamedura">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Duração</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="gameid">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>ID</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="gamename">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Nome</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLineEdit" name="gamegender">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Gênero</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <spacer name="verticalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="2" column="0">
                 <widget class="QLineEdit" name="platgame">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Plataforma</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_12">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <widget class="QPushButton" name="pbcgame">
                  <property name="text">
                   <string>Cadastrar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbsgame">
                  <property name="text">
                   <string>Selecionar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbagame">
                  <property name="text">
                   <string>Atualizar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbegame">
                  <property name="text">
                   <string>Excluir</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_11">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pg_user">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLineEdit" name="userid">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>ID</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="username">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Nome</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLineEdit" name="userage">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Idade</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLineEdit" name="usergender">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>Gênero</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QPushButton" name="pbcuser">
                  <property name="text">
                   <string>Cadastrar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbsuser">
                  <property name="text">
                   <string>Selecionar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbauser">
                  <property name="text">
                   <string>Atualizar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbeuser">
                  <property name="text">
                   <string>Excluir</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="footer_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>BackDB</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
